### PR TITLE
Models

### PR DESCRIPTION
**Pull request by gonjavi**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+    validates :username, uniqueness: true
```
Comment (azdrenymeri) [inline app/models/user.rb]: - [ ] username is validated (at least for the presence)

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    validates :email, uniqueness: true, presence: true, email: true
+    validates :name, presence: {string: true}, length:{minimum: 5}
```
Comment (azdrenymeri) [inline app/models/user.rb]: `[OPTIONAL]` 
 - [ ] making the name of minimum 5 characters I think is not a good idea. 
What if John wants to create an account or Joe or Nick :smile:

File: app/models/user.rb (hunk 1 above)
Comment (gonjavi) [reply app/models/user.rb]: Perfect I changed it for two

File: app/models/user.rb (hunk 2 above)
Comment (gonjavi) [reply app/models/user.rb]: Perfect I changed it for two

File: app/models/user.rb (hunk 1 above)
Comment (gonjavi) [reply app/models/user.rb]: perfect

**Pull request merged**